docs/sphinx/source/singlediode.rst

```diff
@@ -0,0 +1,104 @@
+.. _singlediode:
+
+Single Diode Model
+==================
+
+This section reviews the solutions to the single diode model used in
+pvlib-python to generate an IV curve of a PV module.
+
+pvlib-python supports two ways to solve the single diode model, by passing the
+a ``method`` keyword to the :func:`pvlib.pvsystem.singlediode` function:
```

Comment:
I suggest something like...

pvlib-python supports two ways to solve the single diode model:

1. 
2.

The :func:`pvlib.pvsystem.singlediode` function allows the user to choose the method using the ``method`` keyword.

Reply:
Agree with @wholmgren here

Reply:
✅ nice change thx!